app/addons/documents/index-results/components/pagination/TableControls.js

```diff
@@ -0,0 +1,64 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
All our other component files are all lowercase. So this should be `tablecontrols`. Can you change that for all the files. Otherwise its going to get a bit messy